Update Cloud&Heat Technologies and VP10

mbuechse opened the pull request:
I stumbled upon the team VP10 (hadn't known about it!) and noticed that it was incomplete. So I updated the manager data to the extent that I could.

anjastrunk reviewed (changes requested):
Josephine is missing. You is ch-link?
IMO, Matthias can be added a manager, too.

mbuechse commented:
@anjastrunk thanks for spotting these issues!
I added josephineSei and markus-hentsch, and I moved myself to maintainer as per your suggestion.
cah-link used to work on vp13; he is not on vp10.

garloff reviewed (approved):
/lgtm, thanks!
